libraries/AP_RangeFinder/AP_RangeFinder_JRE_Serial.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "AP_RangeFinder.h"
+#include "AP_RangeFinder_Backend_Serial.h"
+
+#if AP_RANGEFINDER_JRE_SERIAL_ENABLED
+
+#define DATA_LENGTH 16
```

Comment:
I think we should prefix this definition with something like AP_RANGEFINDER_JRE_ (or similar) because this definition is in the global scope and because it is quite generic it could be re-used by another driver.  There are no conflicts at the moment but normally we add a prefix I think.

Reply:
I corrected it.
thank you.